packages/salesforcedx-vscode-core/src/commands/templates/forceFunctionCreate.ts

```diff
@@ -44,6 +47,16 @@ export class ForceFunctionCreateExecutor extends BaseTemplateCommand {
       .withLogName('force_create_function')
       .build();
   }
+
+  public runPostCommandTasks(outputDir: string) {
+    if (sfdxCoreSettings.getFunctionsPullDependencies()) {
+      cp.exec('npm install', { cwd: outputDir }, err => {
```

Comment:
We should also handle the scenario where NodeJS is not present to show the correct error message